backend/pkg/api/connect/service/license/service.go

```diff
@@ -135,12 +135,18 @@ func (s Service) ListLicenses(ctx context.Context, _ *connect.Request[v1alpha1.L
 		}
 	}
 
+	isViolation := false
+	enterpriseFeatures, err := s.adminapiCl.GetEnterpriseFeatures(ctx)
+	if err == nil {
+		isViolation = enterpriseFeatures.Violation
+	}
```

Comment:
if `GetEnterpriseFeatures` fails we default to "no violation"... is that intentional? 
Couldn't just someone turn off / ensure this endpoint fails and always have a valid license?

Reply:
Yes this is intentional, since we will also be dealing with Redpanda clusters that do not yet implement this new endpoint. It is not a problem, because this is just some soft nagging / reminding the users that they are violating the license. If we are not certain they are violating the license we better be quiet was the idea here